.storybook/preview.js

```diff
@@ -2,6 +2,9 @@ import {addons} from '@storybook/addons'
 import {withThemeProvider, withSurroundingElements, toolbarTypes} from '../src/utils/story-helpers'
 import {PrimerBreakpoints} from '../src/utils/layout'
 
+import '@primer/css/dist/primitives.css'
```

Comment:
I'm a little concerned about add a dep for PCSS even though I know this is experimental. I think it makes more sense to just go ahead and start using the new tokens. The size tokens (imported here through primitives.css) will be rolled out to dotcom this week or next (like removing the feature flag) and the colors will start to be tested soon. Since this is a test, I think just go ahead and use the new color tokens!

Here are the imports (but from dist of course) https://github.com/primer/primitives/blob/main/docs/storybook/.storybook/preview.css

I'll be cutting a new release probably this week

Reply:
Made v8 the default now with a fallback on v7. We can remove it as soon as v8 is rolled out in dotcom :)